src/k8s-engine/lima.ts

```diff
@@ -899,6 +923,9 @@ ${ commands.join('\n') }
     console.log(`need to limactl sudoers, get data from ${ tmpFile }`);
     commands.push(`cp "${ tmpFile }" ${ LIMA_SUDOERS_LOCATION } && rm -f "${ tmpFile }"`);
     explanations.push(LIMA_SUDOERS_LOCATION);
+    if (haveFiles[PREVIOUS_LIMA_SUDOERS_LOCATION]) {
+      commands.push(`rm -f ${ PREVIOUS_LIMA_SUDOERS_LOCATION }`);
```

Comment:
Given that it's `-f`, it doesn't hurt to always include this; however, I don't see any expected (read: non-malicious) cases where this will matter.

Reply:
Not sure what you mean, whether you're questioning deleting the old file, or including a `-f`. Unless I care about failure, I usually include a `-f`, even when I'm running as root (which is the case here), just to show that failure can be ignored.

Reply:
He means that you don't need to check if the file exists; you can always add the `rm` command because it will fail silently when the file isn't there.

Reply:
Right, but it's not a matter of replacing
```
if foo exists {
  rm foo
}
```
with `rm -f foo`.

It's a matter of replacing
```
if foo exists {
  bash -c '... ; rm foo; ...'
}
```
with `bash -c '... rm -f foo; ...'

because if we know `foo` doesn't exist we don't have to schedule the command. Seems like a worthwhile savings to me.